java/src/jmri/jmrit/roster/Roster.java

```diff
@@ -321,6 +321,23 @@ public RosterEntry getEntryForId(String id) {
         return null;
     }
 
+     /**
+     * Return a list of RosterEntry which have a 
+     * particular DCC address.
+     *
+     * @param a 
+     * @return an ArrayList of matching entries
+     */
+    public List<RosterEntry> getEntriesByDccAddress(String a) {
+        List<RosterEntry> list = new ArrayList<>(); 
```

Comment:
I'm not sure I agree. The end result is the same but matchingList() does an awful lot of work which is completely unnecessary for no additional benefit. Unless there's something I'm missing.